acceptance/clients/clients.go

```diff
@@ -136,6 +137,23 @@ func NewBlockStorageV2Client() (*gophercloud.ServiceClient, error) {
 	})
 }
 
+// NewBlockStorageV2NoAuthClient returns a noauth *ServiceClient for
+// making calls to the OpenStack Block Storage v2 API. An error will be
+// returned if client creation was not possible.
+func NewBlockStorageV2NoAuthClient() (*gophercloud.ServiceClient, error) {
+	client, err := noauth.NewClient(gophercloud.AuthOptions{
+		Username:   os.Getenv("OS_USERNAME"),
+		TenantName: os.Getenv("OS_TENANT_NAME"),
+	})
+	if err != nil {
+		return nil, err
+	}
+
+	return noauth.NewBlockStorageV2(client, noauth.EndpointOpts{
+		CinderEndpoint: os.Getenv("CINDER_ENDPOINT"),
```

Comment:
@tchughesiv I think OpenStack calls this CINDERCLIENT_BYPASS_URL and, if so, we'll want to use that. Can you confirm or disprove?

Reply:
It's a little convoluted, but I _think_ `CINDER_ENDPOINT` will work:

https://github.com/openstack/python-cinderclient/blob/fd0a11e7d7bec10a18b8bc0b209e5620fca61590/cinderclient/shell.py#L209-L229

Both `CINDERCLIENT_BYPASS_URL` and `CINDER_ENDPOINT` ultimately get stored in `os_endpoint`.

Reply:
It appears CINDERCLIENT_BYPASS_URL has been depreciated in favor of CINDER_ENDPOINT.
https://docs.openstack.org/python-cinderclient/pike/cli/details.html

Reply:
Actually, `CINDER_ENDPOINT` is the best choice. `CINDERCLIENT_BYPASS_URL` is marked as deprecated. `--os-endpoint` is the non-deprecated option, which `CINDER_ENDPOINT` is used as the environment variable.

Reply:
@tchughesiv yep, agreed. arg, I guess Github doesn't automatically show updates for inline threads - just comments.

Reply:
Fine by me